src/main/resources/swagger.api/schemas/d2ir/patronHoldDTO.json

```diff
@@ -36,11 +36,22 @@
     "needBefore": {
       "description": "Need before time",
       "type": "integer"
+    },
+    "patronName": {
+      "description": "Patron name",
+      "type": "string",
+      "maxLength": 255
+    },
+    "centralPatronType": {
+      "description": "Central patron type",
+      "type": "integer",
+      "minimum": 0,
+      "maximum": 255
```

Comment:
these fields are not part of API specification for patron hold, so they should not be needed here.